portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,39 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+	  response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
+    System.out.println(json + "json request");
+    response.getWriter().println(json);
   }
+
+	private String convertToJson(ArrayList list) {
+		String json = "{";
+    json += "\"revolutionary\": ";
+    json += "\"" + list.get(0) + "\"";
+    json += ", ";
```

Comment:
nit: consistent indentation makes code easier to skim & understand function/component hierarchies